docs/managing-findings.md

```diff
@@ -50,57 +51,24 @@ severity:warning rule:finding-test: Finding test 2
 1:print(1 == 1)
 ```
 
-For more information on writing rules, see [Rule syntax](../writing-rules/rule-syntax/).
+For more information on writing rules, see [Rule syntax](/writing-rules/rule-syntax/).
 
 ## Semgrep CI
 
-[Semgrep CI](../semgrep-ci/overview/), designed to continuously scan commits and builds, improves on Semgrep findings to track the lifetime of an individual finding. A Semgrep CI finding is defined by a 4-tuple:
-
-```
-(rule ID, file path, syntactic context, index)
-```
-
-These pieces of state correspond to:
-
-1. `rule ID`: the rule's ID within the Semgrep ecosystem.
-1. `file path`: the filesystem path where the finding occurred.
-1. `syntactic context`: the lines of code corresponding to the finding.
-1. `index`: an index into identical findings within a file. This is used to disambiguate findings.
```

Comment:
@adamkvitek I actually think this information is very important for users to have when they are trying to figure out what a unique finding is.

Reply:
@chmccreery reverted! https://github.com/returntocorp/semgrep-docs/pull/748/commits/dbe71972020c37d7c5983961f1c97210ae998262

Reply:
I also made a little touch stating that 4-tuple is "In the code" https://github.com/returntocorp/semgrep-docs/pull/748/commits/2c2adb836f1811bcc24565fdddea581b9d96e715